### DISCUSSION TITLE
Установка мотора

### DISCUSSION THREAD

**Discussion by Ozzzone** [visavi/rotor#317] [Q&A] [answered]
Как установить мотор без composer? 
сервер на windows

**Comment by visavi** [comment, marked as answer]
@Ozzzone У вас прямо windows сервер или вы локально ставите?
Если локально, то в open server есть уже встроенный композер
также можно установить по инструкции
https://getcomposer.org/doc/00-intro.md#installation-windows

**Comment by visavi** [comment]
В крайнем случае на сайте есть уже собранный архив, композер не требуется запускать 
https://visavi.net/downs/2031

**Comment by Ozzzone** [comment]
У меня сервер на виндовс, а компосер не устанавливается. Не хотел
разбираться с ним, поэтому буду пробовать уже собранный. Спасибо за ответ
:)
сам бы вряд-ли нашел собранный.

пн, 7 авг. 2023 г., 11:55 Alexander Grigorev ***@***.***>:
